src/openrct2/actions/PlaceParkEntranceAction.cpp

```diff
@@ -51,8 +51,7 @@ GameActions::Result::Ptr PlaceParkEntranceAction::Query() const
     res->Expenditure = ExpenditureType::LandPurchase;
     res->Position = { _loc.x, _loc.y, _loc.z };
 
-    if (!LocationValid(_loc) || _loc.x <= 32 || _loc.y <= 32 || _loc.x >= (gMapSizeUnits - 32)
-        || _loc.y >= (gMapSizeUnits - 32))
+    if (!LocationValid(_loc) || map_is_edge(_loc))
```

Comment:
This is also slightly different. <= 32 is now <32. Will need to check if that is correct

Reply:
No its not. Reverting.